ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala

```diff
@@ -30,6 +30,12 @@ private[committer] object PackageCommitter {
   )
 
   type Step = CommitStep[Result]
+
+  // Orphan, but we can't do much about that.
+  // Adding this to Daml-LF would probably be overkill.
+  private implicit val `DamlLf.Archive to LoggingValue`
+      : LoggingValue.ToLoggingValue[DamlLf.Archive] =
+    value => LoggingValue.OfString(value.getHash)
```

Comment:
Not a big problem but I would probably make it explicit on the call site that we're logging the hash.

Reply:
Deleted this in favor of just `.map(_.getHash)` in the log instruction.